app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    if command.startswith("cm"):
+        command = command.replace("cm ", "")
+    else:
+        print("Unknown command")
+    list_of_files_name = command.split(" ")
```

Comment:
Ensure that the command string contains exactly three elements: 'cp', source filename, and destination filename. This is required by the task checklist.